Lib/email/utils.py

```diff
@@ -351,3 +324,11 @@ def localtime(dt=None, isdst=None):
     if dt is None:
         dt = datetime.datetime.now()
     return dt.astimezone()
+
+
+def __getattr__(attr):
+    # lazy import, to speedup module import time
+    if attr == "make_msgid":
+        from ._msgid import make_msgid
```

Comment:
The other intra-package imports (at the top-level) use the spelled-out ``email.spam`` form:
```suggestion
        from email._msgid import make_msgid
```